### ISSUE TITLE
Launch Server invalid Path name

### ISSUE THREAD

**Issue by natedavis** [BF2Statistics/ControlCenter#2]
System: Server 2008 R2 64bit

BF2 is installed:
C:\Program Files (x86)\EA Games\Battlefield 2 Server\
BF2Statistics is install:
C:\Program Files (x86)\BF2Statistics\BF2Statistics Control Center\

When I try and Launch the game with "Use Global Settings File" The Server will not launch with my settings.

From the command line, I tested the following...

Path Generated in MainForm.cs on line 457 - Which is not working:
```
bf2_w32ded.exe +modPath mods/bf2 +config C:\\Program Files (x86)\\BF2Statistics\\BF2Statistics Control Center\\Python\\GlobalServerSettings.con
```
If I use the following to Launch the server manually it works great:
```
bf2_w32ded.exe +modPath mods/bf2 +config "C:\Program Files (x86)\BF2Statistics\BF2Statistics Control Center\Python\GlobalServerSettings.con"
```
Quotes at the end and beginning with no escaping \s. Must be something with the (x86) in the folder name, but unsure.

**Comment by wilson212** [comment]
WIll fix this is the next update :)

**Comment by wilson212** [comment]
Fixed in version 1.10
